src/node.h

```diff
@@ -214,6 +214,18 @@ NODE_EXTERN void Init(int* argc,
                       int* exec_argc,
                       const char*** exec_argv);
 
+class ArrayBufferAllocatorBase : public v8::ArrayBuffer::Allocator {
+ public:
+  virtual ~ArrayBufferAllocatorBase() {}
+  inline uint32_t* zero_fill_field() { return &zero_fill_field_; }
+
+ protected:
+  uint32_t zero_fill_field_ = 1;  // Boolean but exposed as uint32 to JS land.
+};
```

Comment:
Is the intention that we expose `zero_fill_field()` for public use? I don’t think so. It might be easier/expose less internal API surface to just forward-declare `ArrayBufferAllocator` and add an `ArrayBufferAllocator*` parameter to `CreateIsolateData()`?

Reply:
You are right, no public use of zero_fill_field. Just updated it as your suggestion.